modules/storages/app/common/storages/peripherals/connection_validators/nextcloud/ampf_configuration_validator.rb

```diff
@@ -56,6 +57,23 @@ def userless_access_denied
             end
           end
 
+          def group_folder_app_checks
+            required_version = SemanticVersion.parse(
+              nextcloud_dependencies.dig("dependencies", "group_folders_app", "min_version")
+            )
+
+            capabilities = Registry["nextcloud.queries.capabilities"].call(storage: @storage, auth_strategy: noop).result
+            app_name = I18n.t("storages.dependencies.nextcloud.group_folders_app")
+
+            if capabilities.group_folder_disabled?
+              fail_check(:group_folder_app, message(:missing_dependencies, dependency: app_name))
+            elsif capabilities.group_folder_version < required_version
+              fail_check(:group_folder_app, message(:app_version_mismatch))
```

Comment:
:red_circle: This seems to be the wrong translation. As far as I can tell you want to use `group_folder_version_mismatch`. The translation currently used would indicate that the OP integration app has the wrong version.